build.gradle

```diff
@@ -18,9 +19,10 @@ tag {
 
 subprojects {
     apply plugin: 'java'
-    apply plugin: 'idea'
     apply plugin: 'jacoco'
-    apply plugin: "com.srcclr.gradle"
+    apply plugin: 'com.srcclr.gradle'
+    apply plugin: 'maven-publish'
+    apply plugin: 'com.jfrog.bintray'
```

Comment:
Ya, this sort of mixes the new and old approach. The top of the root build.gradle file contains the new plugin declarations: 
```gradle
plugins {
    id "org.sonarqube" version "2.6.2"
    id "com.srcclr.gradle" version "3.0.3" apply false
    id "com.jfrog.bintray" version "1.8.4" apply false
}
```
It specifies `apply false` to two of the plugins so that they are not applied to the root project, however the plugins are retrieved and still added to the gradle path.
In the `subproject{}` the plugins are applied using the old style `apply plugin: 'com.jfrog.bintray'` This works because the plugins have already been added to the gradle path so the `buildscript {}` section is not needed.